sktime/performance_metrics/forecasting/probabilistic/_classes.py

```diff
@@ -256,7 +256,7 @@ def _check_consistent_input(self, y_true, y_pred, multioutput):
         if not isinstance(y_pred, pd.DataFrame):
             raise ValueError("y_pred should be a dataframe.")
 
-        if not all(y_pred.dtypes == float):
+        if not all(y_pred.dtypes == float) and not all(y_pred.dtypes == "Float64"):
```

Comment:
Should it be "or" here?

Reply:
That's what I thought initially but because it is `not` it has to be `and` (I think). I tested both and `or` didn't give expected behaviour. So it only raises the error if all of the dtypes are NOT float AND all of the dtypes are NOT Float64 (OR with NOTs give weird behaviour because it will always be true that they are NOT one or the other).

Reply:
I think it should be `not any(y_pred.dtypes not in [float, Float64])`, or the condition will erroneously fire if some columns are `Float64` and the remaining ones are `float`.

An easier way, I think, is using `is_numeric_dtype`?

Reply:
Ok, that makes sense - will use `is_numeric_dtype`